examples/pytorch_surrogate_model/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _examples-ml-surrogate:
```

Comment:
This file still needs to be linked into the table of contents (TOC) of `docs/source/usage/examples.rst`

Reply:
done!